common/starbase_modules/HFE_TSP_drone_star_base_module.txt

```diff
@@ -0,0 +1,130 @@
+#
+
+hfe_starbase_module_trading_hub = {
+	icon = "GFX_starbase_offworld_trading_company"
+	section = "REFINERY_STARBASE_SECTION"
+	construction_days = 1
+	initial = yes
+
+	potential = {
+		solar_system = {
+			has_star_flag = hfe_drone_economic_zone
+		}
+	}
+
+	possible = {
+		owner = {
+			OR = {
+				is_country_type = hfe_drone
+				is_country_type = drone
+			}
+		}
+		owner = {
+			NOT = {
+				is_country_type = default
+			}
+		}
+	}
+
+	resources = {
+		category = starbase_modules
+		cost = {
+			rare_crystals = 500
+			energy = 1000
+			alloys = 1000
+			influence = 50
+		}
+		upkeep = {
+			energy = 1
+		}
+		upkeep = {
+			trigger = {
+				owner = {
+					NOT = {
+						is_country_type = hfe_drone
+						is_country_type = drone
+						is_country_type = human_fallen_empire
+						is_country_type = fallen_empire
+						is_country_type = awakened_fallen_empire
+						is_country_type = dormant_marauders
+						is_country_type = awakened_marauders
+						is_country_type = ruined_marauders
+						is_country_type = marauder_raiders
+						is_country_type = ratlings
+					}
+					has_technology = tech_mine_rare_crystals
+				}
+			}
+			rare_crystals = 2
+		}
+		upkeep = {
+			trigger = {
+				owner = {
+					NOT = {
+						is_country_type = hfe_drone
+						is_country_type = drone
+						is_country_type = human_fallen_empire
+						is_country_type = fallen_empire
+						is_country_type = awakened_fallen_empire
+						is_country_type = dormant_marauders
+						is_country_type = awakened_marauders
+						is_country_type = ruined_marauders
+						is_country_type = marauder_raiders
```

Comment:
marauders have no economy and ai, why its here

Reply:
They do when they wake up, and so does the Khaganate. The dormant versions are included so that the building doesn't delete it self (like Solar Panels do for non-Gestalt empires).